.github/dependabot.yml

```diff
@@ -86,8 +86,8 @@ updates:
       - dependency-name: org.apache.maven.plugins:*
       - dependency-name: org.codehaus.mojo:*
       - dependency-name: io.fabric8:docker-maven-plugin
-      - dependency-name: net.revelc.code.formatter:formatter-maven-plugin
-      - dependency-name: net.revelc.code:impsort-maven-plugin
+      - dependency-name: org.openrewrite.maven:rewrite-maven-plugin
+      - dependency-name: com.diffplug.spotless:spotless-maven-plugin
```

Comment:
- https://github.com/openrewrite/rewrite/issues/4578

it would be great to only use rewrite and include eclipse format, currently implemented with `formatter-maven-plugin/spot`, into rewrite by supporting new format.

- https://github.com/ndimiduk/spring-framework/blob/master/eclipse-code-formatter.xml

@timtebeek